wundergraph_cli/src/print_schema/mod.rs

```diff
@@ -316,7 +420,7 @@ mod tests {
         println!("Started server");
 
         let client = reqwest::Client::new();
-        std::thread::sleep(std::time::Duration::from_secs(1));
+        std::thread::sleep(std::time::Duration::from_secs(5));
```

Comment:
5 secs was used to run mysql tests locally. But since [ci postgres test](https://github.com/weiznich/wundergraph/runs/772402645) failed for same reason, I increased the sleep duration globally.